rest/resource-server/src/docs/asciidoc/releases.adoc

```diff
@@ -376,3 +376,19 @@ include::{snippets}/should_document_get_usedbyresource_for_release/curl-request.
 
 ===== Example response
 include::{snippets}/should_document_get_usedbyresource_for_release/http-response.adoc[]
+
+[[resources-release-get-release-vulnerabilities]]
+==== Listing release vulnerabilities
+
+A `GET` request will get vulnerabilities of a single release.
+
+include::{snippets}/should_document_get_release_vulnerabilities/links.adoc[]
```

Comment:
Please add this line under line no. 394 with a "===== Links" heading, similar to the other sections in this doc